app/presenters/menu/default_menu.rb

```diff
@@ -84,6 +84,7 @@ def clouds_menu_section
           Menu::Item.new('orchestration_stack', N_('Stacks'),             'orchestration_stack', {:feature => 'orchestration_stack_show_list'},   '/orchestration_stack/show_list'),
           Menu::Item.new('auth_key_pair_cloud', N_('Key Pairs'),          'auth_key_pair_cloud', {:feature => 'auth_key_pair_cloud_show_list'},   '/auth_key_pair_cloud/show_list'),
           Menu::Item.new('placement_group',     N_('Placement Groups'),   'placement_group',     {:feature => 'placement_group_show_list'},       '/placement_group/show_list'),
+          Menu::Item.new('resource_pool_cloud', N_('Resource Pools'),     'resource_pool_cloud', {:feature => 'resource_pool_show_list'},         '/resource_pool_cloud/show_list'),
```

Comment:
I changed the identifier name to Menu::Item.new('resource_pool_cloud') for the cloud section, and for the `infrastructure_menu_section`, I changed it to Menu::Item.new('resource_pool_infra'). After clicking on these two menus, I noticed that neither of them gets highlighted, although they function correctly and display data. Is it also necessary to make similar changes in the `miq_product_features.yml` file?

Reply:
@jeffibm ^